packages/dataviews/src/view-table.js

```diff
@@ -161,7 +153,19 @@ function HeaderMenu( { field, view, onChangeView } ) {
 								<DropdownSubMenuTrigger
 									prefix={ <Icon icon={ funnel } /> }
 									suffix={
-										<Icon icon={ chevronRightSmall } />
+										<>
```

Comment:
Adds the summary ("Is Admin") to the UI:

<img width="226" alt="Captura de ecrã 2023-12-14, às 13 07 05" src="https://github.com/WordPress/gutenberg/assets/583546/0af5f107-f837-4f8f-a906-4b8614359a7d">